scripts/integration_test.sh

```diff
@@ -17,14 +17,29 @@
 # Setup environment for CI.
 set -e
 
-dir=`dirname $0`
+readonly dir=`dirname $0`
+
+GetStatusOfContainerBuild () {
+  local status=$(echo $1 | sed "s/.* SUCCESS$/OK/")
+  if [ $status = "OK" ]; then
+    echo "PASSED"
+  else
+    echo "FAILED"
+  fi
+}
 
 imageUnderTest=$1
 if [ -z "${imageUnderTest}" ]; then
   echo "Usage: ${0} <image_under_test>"
   exit 1
 fi
 
-${dir}/ae_integration_test.sh ${imageUnderTest}
-${dir}/gke_integration_test.sh ${imageUnderTest}
+AE_OUTPUT=$(${dir}/ae_integration_test.sh ${imageUnderTest})
+GKE_OUTPUT=$(${dir}/gke_integration_test.sh ${imageUnderTest})
+
+AE_STATUS=$(GetStatusOfContainerBuild "$AE_OUTPUT")
+GKE_STATUS=$(GetStatusOfContainerBuild "$GKE_OUTPUT")
+
+echo "App Engine integration tests: $AE_STATUS"
+echo "Container Engine integration tests: $GKE_STATUS"
```

Comment:
Also, I think we want to make sure the script return non-zero exit code, if either of the tests failed.